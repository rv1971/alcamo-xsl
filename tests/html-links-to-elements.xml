<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:xh="http://www.w3.org/1999/xhtml"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
      xml:id="xslt"
      version="1.0"
      exclude-result-prefixes="a xh">
    <xsl:import href="../xsl/html.xsl"/>

    <xsl:template match="/">
      <body xmlns:dc="http://purl.org/dc/terms/">
        <p id="P1" xml:id="p1">Paragraph 1</p>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:a"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]/@id" mode="a:a"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]/@xml:id" mode="a:a"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:h2"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:h3"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:h4"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:h5"/>

        <xsl:apply-templates select="//xh:body/xh:p[1]" mode="a:h6"/>
      </body>
    </xsl:template>
  </xsl:stylesheet>
</a:doc>
