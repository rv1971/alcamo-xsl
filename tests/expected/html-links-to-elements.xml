<?xml version="1.0"?>
<body xmlns:dc="http://purl.org/dc/terms/" xmlns="http://www.w3.org/1999/xhtml" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
  <p id="P1" xml:id="p1" rdfs:label="The very first paragraph">Paragraph 1</p>
  <p id="P2">Paragraph 2</p>
  <a href="#p1" title="The very first paragraph">Paragraph 1</a>
  <li>
    <a href="#p1" title="The very first paragraph">Paragraph 1</a>
  </li>
  <a href="#P1" title="The very first paragraph">P1</a>
  <li>
    <a href="#P1" title="The very first paragraph">P1</a>
  </li>
  <a href="#p1" title="The very first paragraph">p1</a>
  <li>
    <a href="#p1" title="The very first paragraph">p1</a>
  </li>
  <h2 id="p1">Paragraph 1</h2>
  <a href="#P2">P2</a>
  <h3 id="P2">Paragraph 2</h3>
  <h4 id="P2">Paragraph 2</h4>
  <h5 id="P2">Paragraph 2</h5>
  <h6 id="P2">Paragraph 2</h6>
</body>
